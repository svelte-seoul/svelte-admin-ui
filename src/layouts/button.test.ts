import Button from './button.svelte';
import {render} from '@testing-library/svelte';

it('should render without error', () => {
  const testngLib = render(Button);

  expect(testngLib.container).toMatchSnapshot();
});
