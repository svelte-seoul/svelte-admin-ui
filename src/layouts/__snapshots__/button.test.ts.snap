// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without error 1`] = `
<body>
  <div>
    <button
      class="svelte-1a5mybq medium"
      style=""
    />
  </div>
</body>
`;
